Resources/Prototypes/Entities/Structures/Storage/Tanks/tanks.yml

```diff
@@ -157,3 +157,68 @@
     solution: tank
   - type: ReagentTank
     transferAmount: 100
+
+    # WeedKiller
+
+- type: entity
+  id: WeedKillerTank
+  parent: StorageTank
+  suffix: Empty
+  components:
+  - type: StaticPrice
+    price: 500
+  - type: Sprite
+    sprite: Structures/Storage/tanks.rsi
+    layers:
+      - state: watertank-2
+      - state: watertank-2-1
+        map: ["enum.SolutionContainerLayers.Fill"]
+        visible: false
+  - type: Appearance
+  - type: SolutionContainerVisuals
+    maxFillLevels: 5
+    fillBaseName: watertank-2-
+  - type: ExaminableSolution
+    solution: tank
+
+- type: entity
+  parent: WeedKillerTank
+  id: WeedKillerTankFull
+  suffix: Full
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      tank:
+        reagents:
+        - ReagentId: WeedKiller
+          Quantity: 1500
+
+- type: entity
+  parent: StorageTank
+  id: WeedKillerTankHighCapacity
+  suffix: Full
+  components:
+  - type: Sprite
+    sprite: Structures/Storage/tanks.rsi
+    layers:
+      - state: highwatertank-2
+      - state: highwatertank-2-1
+        map: ["enum.SolutionContainerLayers.Fill"]
+        visible: false
+  - type: Appearance
+  - type: SolutionContainerVisuals
+    maxFillLevels: 7
+    fillBaseName: highwatertank-2-
+  - type: SolutionContainerManager
+    solutions:
+      tank:
+        reagents:
+        - ReagentId: WeedKiller
+#         This is *100, 000* on /tg/
+          Quantity: 10000
+        maxVol: 10000
```

Comment:
This is enough to get 500 puddles so probably should be capped to 1 or 2 k, look at water / fuel.

Reply:
Whoops, I should have been more clear.
This PR adds the regular tank size (1500u) and the high capacity tank (10000u) for completeness.
However only the regular size tank is available to cargo.

You're looking at the High Capacity tank, its not available anywhere but the entity spawn panel.